pkg/gce-pd-csi-driver/utils.go

```diff
@@ -59,3 +62,47 @@ func logGRPC(ctx context.Context, req interface{}, info *grpc.UnaryServerInfo, h
 	}
 	return resp, err
 }
+
+func validateVolumeCapabilities(vcs []*csi.VolumeCapability) error {
+	if vcs == nil {
+		return errors.New("volume capabilities is nil")
+	}
+	for _, vc := range vcs {
+		if err := validateVolumeCapability(vc); err != nil {
+			return err
+		}
+	}
+	return nil
+}
+
+func validateVolumeCapability(vc *csi.VolumeCapability) error {
+	if err := validateAccessModes(vc.GetAccessMode()); err != nil {
+		return err
+	}
+	if blk := vc.GetBlock(); blk != nil {
```

Comment:
can both block and mount be set at the same time?

Reply:
I don't believe so: https://github.com/container-storage-interface/spec/blame/master/spec.md#L855